shared/js/background/chrome-events.es6.js

```diff
@@ -201,6 +201,46 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
     }
 })
 
+/**
+ * Fingerprint Protection
+ */
+
+// Inject fingerprint protection into sites when
+// they are not whitelisted.
+chrome.webNavigation.onCommitted.addListener(details => {
+    const activeExperiment = settings.getSetting('activeExperiment')
+
+    if (activeExperiment) {
+        const experiment = settings.getSetting('experimentData')
+
+        if (experiment && experiment.fingerprint_protection) {
+            const whitelisted = settings.getSetting('whitelisted')
+            const tabURL = new URL(details.url) || {}
+            if (!whitelisted || !whitelisted[tabURL.hostname]) {
+                const scriptDetails = {
+                    'file': '/data/fingerprint-protection.js',
+                    'runAt': 'document_start',
+                    'allFrames': true
```

Comment:
It occurred to me that the issue that you mentioned with scirpt not being injected into certain iframes might be because they don't have any 'src' (e.g. because they were dynamically crated). This should be fixed by adding `matchAboutBlank: true` here. Could you please verify if this addresses the issue?

Reply:
It does seem to work, so the other iframe injections can be removed.

However after this, the iframes don't seem to receive screen size updates. This doesn't seem to cause any issues in testing.

Reply:
> It does seem to work, so the other iframe injections can be removed.

That's great, thank you for verifying - it will make the code simpler 🙌

> don't seem to receive screen size updates.

Hm, not sure what that's about, regular iframes seem to receive 'resize' events - might be a Chrome bug. I'll take a look when testing.

Reply:
When I played with the extension I saw some frames that are not receiving `resize` events, but I was able to reproduce this with and without the extension, so it doesn't seem to be related.